app/src/main/java/com/sdp13epfl2021/projmag/database/LocalFileUtils.kt

```diff
@@ -3,47 +3,46 @@ package com.sdp13epfl2021.projmag.database
 import java.io.*
 import kotlin.reflect.KClass
 
-object LocalFileUtils {
 
-    /**
-     * Save a serializable object to a file.
-     *
-     * @param file The file where data will be stored.
-     * @param data The parcelable data to store.
-     * @return if the operation was successful.
-     */
-    fun saveToFile(file: File, data: Serializable) : Boolean {
-        try {
-            ObjectOutputStream(FileOutputStream(file, false)).use {
-                it.writeObject(data)
-                return true
-            }
-        } catch (e: Exception) {
-            e.printStackTrace()
-            return false
+/**
+ * This serializable class can be used to wrap a list of String.
+ */
+data class SerializedStringListWrapper(val list: List<String>) : Serializable
```

Comment:
why is this class added ?

Reply:
It was in OfflineCachedUserDataDatabase.
A List<*> is not serializable directly, so I use this class to wrap a list of string. 
Because it could be used in other offline databases/etc, I move it to LocalFileUtils.